index.rst

```diff
@@ -47,7 +46,8 @@ non-root user.
 signature travels with the container image, allowing you to verify that
 the image is unmodified at any time. Encryption of containers using
 LUKS2 is also supported. Encrypted containers can be run without
-decrypting them to disk first.
+decrypting them to disk first when using a setuid installation of
```

Comment:
I would be careful with wording here to make sure we don't imply that non-setuid installations decrypt content to disk first. Maybe:

Encrypted containers can be run without decrypting them to disk first. Note: Execution of encrypted containers currently requires a setuid installtion.